python/graphstorm/model/rgcn_encoder.py

```diff
@@ -233,9 +260,15 @@ def forward(self, g, inputs):
                     inputs_dst[k] = th.zeros((g.num_dst_nodes(k), self.in_feat),
                                              dtype=th.float32, device=g.device)
         else:
-            inputs_src = inputs_dst = inputs
+            inputs_src = inputs_dst = n_h
 
-        hs = self.conv(g, (inputs_src, inputs_dst), mod_kwargs=wdict)
+        if e_h is None:
```

Comment:
It will trigger a pylint `dangerous-default-value` error. This implementation is suggested by pylint: https://pylint.pycqa.org/en/latest/user_guide/messages/warning/dangerous-default-value.html